gen3/bin/kube-setup-networkpolicy.sh

```diff
@@ -49,6 +49,7 @@ if [[ -f "$credsPath" ]]; then # setup netpolicy
   g3k_kv_filter "${GEN3_HOME}/kube/services/netpolicy/networkpolicy_google-sa-validation_templ.yaml" GEN3_FENCEDB_IP "$FENCEDB_IP" GEN3_CLOUDPROXY_CIDR "$CLOUDPROXY_CIDR" | g3kubectl apply -f -
   g3k_kv_filter "${GEN3_HOME}/kube/services/netpolicy/networkpolicy_indexd_templ.yaml" GEN3_INDEXDDB_IP "$INDEXDDB_IP" GEN3_CLOUDPROXY_CIDR "$CLOUDPROXY_CIDR" | g3kubectl apply -f -
   g3k_kv_filter "${GEN3_HOME}/kube/services/netpolicy/networkpolicy_peregrine_templ.yaml" GEN3_GDCAPIDB_IP "$GDCAPIDB_IP" GEN3_CLOUDPROXY_CIDR "$CLOUDPROXY_CIDR" | g3kubectl apply -f -
+  g3k_kv_filter "${GEN3_HOME}/kube/services/netpolicy/networkpolicy_wts_templ.yaml" GEN3_GDCAPIDB_IP "$GDCAPIDB_IP" GEN3_CLOUDPROXY_CIDR "$CLOUDPROXY_CIDR" | g3kubectl apply -f -
```

Comment:
delete this

Reply:
you mean the variables?

Reply:
oh - I was thinking to remove the `g3k_kv_filter`, since the yaml doesn't have variables in it, but it feeds directly into `g3kubectl apply -f`, so it's fine - sorry for the confusion :+1:

Reply:
I didn't update this line cuz I was waiting for you to provide me ways to specify the db ip, and I can add the egress restriction back :p